### PR TITLE
feat: auth screen mobile responsiveness tm-56

### PR DESCRIPTION
**Pull request by YourTeaChan**
- Sign-in page on laptop screen (<= 1024px)

image

- Sign-in page on tablet screen (<= 768px)

image

- Sign-in page on mobile screen (<= 480px)

image

- Sign-up page on laptop screen (<= 1024px)

image

- Sign-up page on tablet screen (<= 768px)

image

- Sign-up page on mobile screen (<= 480px)

image

### CODE REVIEW

**Review by Ann-Maksay — APPROVED** [review]

File: apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css [hunk 1]
```diff
@@ -30,3 +30,25 @@
 .link:hover {
 	text-decoration: underline;
 }
+
+@media screen and (width <= 500px) {
```
Comment (Oleksandra-Masiuk) [inline apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:34]: Move breakpoints to CSS variables and update all your media queries. 
We will have three breakpoints: 480px, 768px 1024px.

File: apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css [hunk 2]
```diff
@@ -30,3 +30,25 @@
...
+	.form {
+		max-width: unset;
+		padding: 0 40px;
+	}
+
+	.form * {
```
Comment (Oleksandra-Masiuk) [inline apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:40]: B10

File: apps/frontend/src/pages/auth/styles.module.css [hunk 3]
```diff
@@ -81,3 +81,67 @@
...
+	.logo-wrapper {
+		position: unset;
+		column-gap: 24px;
+		justify-content: center;
+		font-size: var(--font-size-700);
+		color: var(--main-white);
+		background-color: unset;
+	}
+
+	.logo-wrapper * {
```
Comment (Oleksandra-Masiuk) [inline apps/frontend/src/pages/auth/styles.module.css:134]: B10

File: apps/frontend/src/pages/auth/components/sign-up-form/styles.module.css [hunk 4]
```diff
@@ -31,3 +31,26 @@
...
 }
+
+@media screen and (width <= 500px) {
+	.form {
+		max-width: unset;
+		padding: 0 40px;
+		margin-top: 0;
+	}
+
+	.form * {
```
Comment (Oleksandra-Masiuk) [inline apps/frontend/src/pages/auth/components/sign-up-form/styles.module.css:42]: B10

**Review by Oleksandra-Masiuk — CHANGES_REQUESTED** [review]

**Review by Sytnikovakatya — APPROVED** [review]

**Review by GEOFARL — APPROVED** [review]

File: apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css (hunk 1 above)
Comment (YourTeaChan) [reply apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:34]: As I understand, I can't use `var` to define screen width in media query
https://github.com/vercel/next.js/discussions/59224

So, can I just write `@media screen and (width <= 1024px)` and use only 480px, 768px and 1024px. Or there is another way to do it?

Comment (Oleksandra-Masiuk) [reply apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:34]: Yes, sorry, I forgot that only preprocessor's variables can be used in the media queries. I guess in this case we should use 480px, 768px, and 1024px directly. @fshabanov @what1s1ove could you please confirm?

Comment (fshabanov) [reply apps/frontend/src/pages/auth/components/sign-in-form/styles.module.css:34]: Yes, let's use directly in this case

File: apps/frontend/src/pages/auth/styles.module.css [hunk 5]
```diff
@@ -81,3 +81,76 @@
...
+			calc(50% - 160px) 90%,
+			calc(50% + 200px) 20%;
+	}
+
+	.form-container {
+		flex-direction: column;
+		justify-content: flex-start;
+		width: 100%;
+		height: 100%;
+		min-height: 100lvh;
```
[the remark this answers is not in the document]
Comment (YourTeaChan) [reply apps/frontend/src/pages/auth/styles.module.css:133]: So, do I need to change 1001vh to something else or not?

[the remark this answers is not in the document]
Comment (Oleksandra-Masiuk) [reply apps/frontend/src/pages/auth/styles.module.css:133]: then you can keep it

**Review by Oleksandra-Masiuk — APPROVED** [review]

**Review by Vfivew — APPROVED** [review]

**Review by fshabanov — APPROVED** [review]

**Pull request merged**